src/app/(auth)/layout.tsx

```diff
@@ -7,7 +7,7 @@ export default function LoginLayout({
   return (
     <>
       <Header />
-      <main className="min-h-[calc(100vh-4rem-4rem)]">{children}</main>
+      <main className="min-h-[calc(100vh-4rem)]">{children}</main>
```

Comment:
Here you're going to need to subtract the height of the footer to allow it to display on the page when there isn't a lot of content -- before the footer was 4rem tall so that's why it was subtracted. Currently the footer is pushed below the screen because the 100vh-4rem allows for the header but then pushes it below the viewport height:
<img width="1013" alt="Screenshot 2025-03-19 at 6 24 00 PM" src="https://github.com/user-attachments/assets/00ba3ed6-646b-47b5-b371-4e995208c9ea" />

<img width="1026" alt="Screenshot 2025-03-19 at 6 24 07 PM" src="https://github.com/user-attachments/assets/98103e38-db78-47c4-88db-c27ef8e804bc" />